model/shared/src/main/scala/explore/modes/SpectroscopyModesMatrix.scala

```diff
@@ -401,7 +401,7 @@ case class SpectroscopyModesMatrix(matrix: List[SpectroscopyModeRow]) {
         wavelength.forall(w => w >= r.λmin.value && w <= r.λmax.value) &&
         resolution.forall(_ <= r.resolution) &&
         range.forall(_ <= r.λdelta.value) &&
-        slitWidth.forall(_.toMicroarcseconds <= r.slitLength.value.toMicroarcseconds) &&
+        slitWidth.forall(_.toMicroarcseconds <= r.slitWidth.value.toMicroarcseconds) &&
```

Comment:
When the slit width is specified, it must be less than the configuration option slit width value?  In other words, in the requirements the slit width entered is the maximum value that the user will accept?

Reply:
I think slit *length* is the desired parameter here:
https://github.com/gemini-hlsw/explore-help-docs/edit/main/configuration/focal_plane.md